geode-core/src/main/resources/org/apache/geode/internal/sanctioned-geode-core-serializables.txt

```diff
@@ -259,6 +259,11 @@ org/apache/geode/distributed/internal/deadlock/MessageDependencyMonitor$MessageK
 org/apache/geode/distributed/internal/direct/ShunnedMemberException,true,-6455664684151074915
 org/apache/geode/distributed/internal/locks/DistributedMemberLock$LockReentryPolicy,false
 org/apache/geode/distributed/internal/locks/LockGrantorDestroyedException,true,-3540124531032570817
+org/apache/geode/distributed/internal/membership/gms/api/MemberDisconnectedException,true,-3649273301807236514
+org/apache/geode/distributed/internal/membership/gms/api/MemberShunnedException,true,-8453126202477831557
+org/apache/geode/distributed/internal/membership/gms/api/MemberStartupException,true,6610743861046044144
+org/apache/geode/distributed/internal/membership/gms/api/MembershipClosedException,true,6112938405434046127
+org/apache/geode/distributed/internal/membership/gms/api/MembershipConfigurationException,true,5633602142465129621
```

Comment:
Just curious - do we want these new exceptions in the sanctioned serializables, or the excluded ones? Do we want them to go over the wire?

Reply:
I thought about that, too.  Putting them in excludedClasses.txt won't keep them from being serialized unless a serialization filter is installed.